CMakeLists.txt

```diff
@@ -81,7 +81,9 @@ foreach(lib screen dom component)
     PRIVATE src
   )
   target_compile_features(${lib} PUBLIC cxx_std_17)
-  target_compile_options(${lib} PRIVATE -Wall -Werror -pedantic -Wextra -Wno-sign-compare)
+  if(NOT MSVC)
```

Comment:
I am not against this change, but could you explain why it is needed?

Reply:
MSVC doesn’t understand those command line arguments, it uses warnings levels /W[1-4] (or you can enable or disable individual warnings using /W[ed]XXXX)

Reply:
Okay it looks good to me.